spec.md

```diff
@@ -1841,6 +1845,48 @@ To disallow infinitely recursive hardware, modules cannot contain instances of
 itself, either directly, or indirectly through instances of other modules it
 instantiates.
 
+## Groups
+
+Groups are named collections of statements inside a module.  Groups contain
+optional functionality for a module that can be included or excluded without
```

Comment:
Given this meaning, should the name of this concept be something like "optional group"?

Reply:
Possibly. I'll switch to that for now. I also considered "verification groups", but that may be too narrow for their use.

Reply:
Switched to "optional groups".